agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/PhenotypeAnnotationCurationIndexer.java

```diff
@@ -0,0 +1,251 @@
+package org.alliancegenome.indexer.indexers.curation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.alliancegenome.api.entity.AllelePhenotypeAnnotationDocument;
+import org.alliancegenome.api.entity.GenePhenotypeAnnotationDocument;
+import org.alliancegenome.api.entity.PhenotypeAnnotationDocument;
+import org.alliancegenome.curation_api.model.entities.*;
+import org.alliancegenome.es.util.ProcessDisplayHelper;
+import org.alliancegenome.indexer.RestConfig;
+import org.alliancegenome.indexer.config.IndexerConfig;
+import org.alliancegenome.indexer.indexers.Indexer;
+import org.alliancegenome.indexer.indexers.curation.service.AGMPhenotypeAnnotationService;
+import org.alliancegenome.indexer.indexers.curation.service.AllelePhenotypeAnnotationService;
+import org.alliancegenome.indexer.indexers.curation.service.GenePhenotypeAnnotationService;
+import org.alliancegenome.indexer.indexers.curation.service.VocabularyService;
+import org.alliancegenome.neo4j.repository.DiseaseRepository;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.*;
+import java.util.Map.Entry;
+import java.util.concurrent.LinkedBlockingDeque;
+
+@Slf4j
+public class PhenotypeAnnotationCurationIndexer extends Indexer {
+
+	private GenePhenotypeAnnotationService geneService = new GenePhenotypeAnnotationService();
+	private AllelePhenotypeAnnotationService alleleService = new AllelePhenotypeAnnotationService();
+	private AGMPhenotypeAnnotationService agmService = new AGMPhenotypeAnnotationService();
+	private VocabularyService vocabService = new VocabularyService();
+	private DiseaseRepository diseaseRepository;
+
+	private Map<String, Set<String>> closureMap;
+	private Map<String, Pair<Gene, ArrayList<PhenotypeAnnotation>>> geneMap = new HashMap<>();
+	private Map<String, Pair<Allele, ArrayList<PhenotypeAnnotation>>> alleleMap = new HashMap<>();
+	private Map<String, Pair<AffectedGenomicModel, ArrayList<PhenotypeAnnotation>>> agmMap = new HashMap<>();
+
+	public PhenotypeAnnotationCurationIndexer(IndexerConfig indexerConfig) {
+		super(indexerConfig);
+	}
+
+	@Override
+	protected ObjectMapper customizeObjectMapper(ObjectMapper objectMapper) {
+		return RestConfig.config.getJacksonObjectMapperFactory().createObjectMapper();
+	}
+
+	@Override
+	protected void startSingleThread(LinkedBlockingDeque<String> queue) {
+
+	}
+
+	@Override
+	protected void index() {
+
+		diseaseRepository = new DiseaseRepository();
+		closureMap = diseaseRepository.getDOClosureChildMapping();
```

Comment:
Not sure why these are needed as they are being pulled from Neo. closureMap is not used in this code at all, I think the closureMap and the DiseaseRepository need to be removed.